src/NuGet.Clients/NuGet.CommandLine/Commands/SearchCommand.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Commands;
+
+namespace NuGet.CommandLine
+{
+    [Command(
+        typeof(NuGetCommand),
+        "search",
+        "SearchCommandDescription",
+        UsageSummaryResourceName = "SearchCommandUsageSummary",
+        UsageDescriptionResourceName = "SearchCommandUsageDescription",
+        UsageExampleResourceName = "SearchCommandUsageExamples")]
+    public class SearchCommand : Command
+    {
+        private readonly List<string> _sources = new List<string>();
+
+        [Option(typeof(NuGetCommand), "ListCommandSourceDescription")]
+        public ICollection<string> Source
+        {
+            get { return _sources; }
+        }
+
+        [Option(typeof(NuGetCommand), "ListCommandVerboseListDescription")]
+        public bool Verbose { get; set; }
```

Comment:
>Verbose [](start = 20, length = 7)

To me `Verbose` suggests debugging the CLI command. Thoughts on renaming this to `Detailed`?

Reply:
This is deprecated. 
Remove it completely. 

A new verbosity option was added.

Reply:
I will remove the -verbose option for the `nuget.exe search` command. 
NOTE: All changes related for `nuget.exe search` will be addressed in another PR

Reply:
That's fine as long as they both make it in the same release cycle :)

Reply:
This feedback will be addressed in https://github.com/NuGet/Home/issues/8040